controllers/item.js

```diff
@@ -19,11 +19,23 @@ item.withFields = (queryBuilder) => {
   // Category
     .leftJoin('category', 'item.categoryID', 'category.categoryID')
     .select('category.name as category')
+
+  // Mapping between query param fields and database query column names
+  const filterParams = {
+    brand: 'brand.brandID',
+    model: 'model.modelID',
+    category: 'category.categoryID'
+  }
```

Comment:
I was thinking that, too. Would you also want the ability to send the names of brands, models, and categories?